src/main/java/org/terasology/genome/GenomeRegistry.java

```diff
@@ -17,23 +17,21 @@
 
 /**
  * Registry used for registering organism types.
- *
- * @author Marcin Sciesinski <marcins78@gmail.com>
  */
 public interface GenomeRegistry {
     /**
      * Registers the organism type.
      *
-     * @param typeId           Unique type of an organism.
-     * @param genomeDefinition Definition of the genome for that organism.
+     * @param typeId           Unique type of an organism
+     * @param genomeDefinition Definition of the genome for that organism
      */
     void registerType(String typeId, GenomeDefinition genomeDefinition);
 
     /**
      * Returns a genome definition for the specified organism type.
      *
-     * @param typeId Type of an organism.
-     * @return GenomeDefinition for that organism type.
+     * @param typeId Type of an organism
+     * @return       GenomeDefinition for that organism type
```

Comment:
I wouldn't worry about whitespace aligning stuff, it tends to get messy and just adds a tiny bit of maintenance needs :-)